addons/mail/static/src/chatter/web_portal/thread_service_patch.js

```diff
@@ -0,0 +1,99 @@
+/* @odoo-module */
+
+import { ThreadService } from "@mail/core/common/thread_service";
+
+import { rpc } from "@web/core/network/rpc";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+import { Record } from "@mail/core/common/record";
+
+patch(ThreadService.prototype, {
+    /**
+     * @param {import("models").Thread} thread
+     * @param {['activities'|'followers'|'attachments'|'messages'|'suggestedRecipients']} requestList
+     */
+    async fetchData(
+        thread,
+        requestList = ["activities", "followers", "attachments", "messages", "suggestedRecipients"]
+    ) {
+        thread.isLoadingAttachments =
+            thread.isLoadingAttachments || requestList.includes("attachments");
+        if (requestList.includes("messages")) {
+            this.fetchNewMessages(thread);
+        }
+        const result = await rpc("/mail/thread/data", {
+            request_list: requestList,
+            thread_id: thread.id,
+            thread_model: thread.model,
+        });
+        thread.canPostOnReadonly = result.canPostOnReadonly;
+        thread.hasReadAccess = result.hasReadAccess;
+        thread.hasWriteAccess = result.hasWriteAccess;
+        if ("activities" in result) {
+            const existingIds = new Set();
+            for (const activity of result.activities) {
+                existingIds.add(this.store.Activity.insert(activity, { html: true }).id);
+            }
+            for (const activity of thread.activities) {
+                if (!existingIds.has(activity.id)) {
+                    this.activityService.delete(activity);
+                }
+            }
+        }
+        if ("attachments" in result) {
+            Object.assign(thread, {
+                areAttachmentsLoaded: true,
+                attachments: result.attachments,
+                isLoadingAttachments: false,
+            });
+        }
```

Comment:
Didn't we not want activity and attachment in portal?

Reply:
According to task ESPC, no